src/components/ReturnButton.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+  <button class="return-btn" v-on:click="this.return">
+    <img class="return-btn" src="../assets/left-arrow.svg" alt="actions" />
+  </button>
+</template>
+
+<script>
+export default {
+  name: 'return-button',
+  methods: {
+    return() {
+      this.$router.back();
```

Comment:
Is this what we want? I think with this if you create a team (and are redirected to the single team view) if you press the back arrow you'd be redirected to the create form instead of the available teams as one might expect. That's just one example but I think we need the ability to pass in an optional parameter for a specifc view to go back to.

Reply:
Sure, we could do that. It just seemed natural to me from a usability standpoint that a back button takes you to the last page you were on. But I see that use case and that should be easy to add

Reply:
Browsers already have a back button for this, the point of adding an additional one in our UI is so that we can go back to the last logical page rather than the exact last page they were on